charts/dapr/charts/dapr_scheduler/templates/dapr_scheduler_service.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
     {{- end }}
   {{- if .Values.service.annotations }}
     {{- range $key, $value := .Values.service.annotations }}
-      {{ $key }}: "{{ $value }}"
+    {{ $key }}: "{{ $value }}"
```

Comment:
Why not using `{{- .Values.service.annotations | toYaml | nindent 4 }}` like in the other palces? Similar comment in other places

Reply:
Thanks, I'll update with that :slightly_smiling_face: